sklearn/datasets/_base.py

```diff
@@ -682,26 +682,29 @@ def load_breast_cancer(*, return_X_y=False, as_frame=False):
         data : {ndarray, dataframe} of shape (569, 30)
             The data matrix. If `as_frame=True`, `data` will be a pandas
             DataFrame.
-        target: {ndarray, Series} of shape (569,)
+        target : {ndarray, Series} of shape (569,)
             The classification target. If `as_frame=True`, `target` will be
             a pandas Series.
-        feature_names: list
+        feature_names : list
             The names of the dataset columns.
-        target_names: list
+        target_names : list
             The names of target classes.
-        frame: DataFrame of shape (569, 31)
+        frame : DataFrame of shape (569, 31)
             Only present when `as_frame=True`. DataFrame with `data` and
             `target`.
 
             .. versionadded:: 0.23
-        DESCR: str
+        DESCR : str
             The full description of the dataset.
-        filename: str
+        filename : str
             The path to the location of the data.
 
             .. versionadded:: 0.20
 
     (data, target) : tuple if ``return_X_y`` is True
+        A tuple of two ndarrays. The first contains a 2D array of shape (569, 30) 
+        with each row representing one sample and each column representing the features. 
+        The second array of shape (569,) contains the target samples.
```

Comment:
```suggestion
        A tuple of two ndarrays. The first contains a 2D array of shape (569, 30) 
        with each row representing one sample and each column representing the features. 
        The second ndarray of shape (569,) contains the target samples.
```
Maybe mentioning again that the target samples are provided as an ndarray could make the description a bit easier to understand. 

Also try to follow the PEP 8 convention, i.e. limit all lines to a maximum of 79 characters and avoid blank spaces at the end of each line.

Reply:
> I'm still getting the "return value has no description" error.

  The following lines of code are creating the error:

```python  
The copy of UCI ML Breast Cancer Wisconsin (Diagnostic) dataset is
downloaded from:
https://goo.gl/U2Uwz2
```

I propose moving them to the description in the header below the table.

Reply:
Thanks for the help @ArturoAmorQ it worked.